docs/Data_Portal/Users_Guide/Repository.md

```diff
@@ -85,6 +106,10 @@ The Cart Items table shows the list of all the files that were added to the Cart
 * **File Size**: The size of the file.
```

Comment:
Same comment above regarding periods at the end of single-sentence bullet-points